src/vm/gdbjit.cpp

```diff
@@ -2473,12 +2487,35 @@ void NotifyGdb::OnMethodPrepared(MethodDesc* methodDescPtr)
 
     elfBuilder.Finalize();
 
+    char   *symfile_addr = NULL;
+    size_t  symfile_size = 0;
+
+    symfile_addr = elfBuilder.Export(&symfile_size);
+
+#if _DEBUG
+    static LPCUTF8 pszGDBJitElfDump = 0;
+
+    if (!pszGDBJitElfDump)
+    {
+        LPWSTR pszGDBJitElfDumpW = NULL;
+        CLRConfig::GetConfigValue(CLRConfig::INTERNAL_GDBJitElfDump, &pszGDBJitElfDumpW);
+        pszGDBJitElfDump = EEConfig::NarrowWideChar(pszGDBJitElfDumpW);
+    }
+
+    LPCUTF8 methodName = methodDescPtr->GetName();
+    if (EEConfig::RegexOrExactMatch(pszGDBJitElfDump, methodName))
```

Comment:
I would prefer adding ShouldDumpElfForMethod(LPCUTF8 methodName) and LPUTF8 pszGDBJitElfDump member to EEConfig to keep it in line with the other settings.